assets/js/googlesitekit/data/utils.test.js

```diff
@@ -124,6 +128,231 @@ describe( 'data utils', () => {
 		} );
 	} );
 
+	describe( 'combineStores()', () => {
+		const combineStoresFakeActionOne = {
+			fakeActionOne() {
+				return { type: 'ACTION_ONE', payload: {} };
+			},
+		};
+
+		const combineStoresFakeActionTwo = {
+			fakeActionTwo() {
+				return { type: 'ACTION_TWO', payload: {} };
+			},
+		};
+
+		const combineStoresFakeReducerOne = ( state, action ) => {
+			switch ( action.type ) {
+				case 'ACTION_ONE':
+					return { ...state, one: true };
+				default: {
+					return { ...state };
+				}
+			}
+		};
+
+		const combineStoresFakeReducerTwo = ( state, action ) => {
+			switch ( action.type ) {
+				case 'ACTION_TWO':
+					return { ...state, two: 2 };
+				default: {
+					return { ...state };
+				}
+			}
+		};
+
+		const combineStoresFakeReducerFour = ( state, action ) => {
+			switch ( action.type ) {
+				case 'ACTION_FOUR':
+					return { ...state, four: 4 };
+				default: {
+					return { ...state };
+				}
+			}
+		};
+
+		const combineStoresFakeControlsOne = {
+			FAKE_CONTROL_ONE: () => {
+				return null;
+			},
+		};
+
+		const combineStoresFakeControlsTwo = {
+			FAKE_CONTROL_TWO: () => {
+				return null;
+			},
+		};
+
+		const combineStoresFakeResolversOne = {
+			*getFakeActionOne() {
+				yield combineStoresFakeActionOne.fakeActionOne();
+			},
+		};
+
+		const combineStoresFakeResolversTwo = {
+			*getFakeActionTwo() {
+				yield combineStoresFakeActionTwo.fakeActionTwo();
+			},
+		};
+
+		const combineStoresFakeSelectorsOne = {
+			getOne: ( state ) => {
+				return state.one;
+			},
+		};
+		const combineStoresFakeSelectorsTwo = {
+			getTwo: ( state ) => {
+				return state.two;
+			},
+		};
+
+		const storeOne = {
+			INITIAL_STATE: { one: 1 },
+			actions: combineStoresFakeActionOne,
+			controls: combineStoresFakeControlsOne,
+			reducer: combineStoresFakeReducerOne,
+			resolvers: combineStoresFakeResolversOne,
+			selectors: combineStoresFakeSelectorsOne,
+		};
+
+		const storeTwo = {
+			INITIAL_STATE: { two: 2 },
+			actions: combineStoresFakeActionTwo,
+			controls: combineStoresFakeControlsTwo,
+			reducer: combineStoresFakeReducerTwo,
+			resolvers: combineStoresFakeResolversTwo,
+			selectors: combineStoresFakeSelectorsTwo,
+		};
+
+		const storeThree = {
+			INITIAL_STATE: { three: 3 },
+		};
+
+		const storeFour = {
+			reducer: combineStoresFakeReducerFour,
+		};
+
+		const storeFive = {
+			actions: {
+				fakeActionOne() {
```

Comment:
Looks like this action is defined above but rewritten here. Seems like we could consolidate the number of things defined here in addition to @aaemnnosttv's comment about colocating the variables closer to their tests.

Reply:
:+1: